SPECS/python-tqdm/python-tqdm.spec

```diff
@@ -0,0 +1,160 @@
+%global srcname tqdm
+
+Summary:        Fast, Extensible Progress Meter
+Name:           python-%{srcname}
+Version:        4.50.2
+Release:        2%{?dist}
+License:        MPLv2.0 and MIT
+URL:            https://github.com/tqdm/tqdm
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Source0:        %{url}/archive/v%{version}/%{srcname}-%{version}.tar.gz
```

Comment:
>%{srcname}-%{version}.tar.gz [](http://example.com/codeflow?start=43&length=28)

Might be wrong, but I think this will make the cgmanifest checker angry. For python packages that don't follow %{name}-%{version}.whatever, you can add them to the exception list in the script.